projects/plugins/jetpack/readme.txt

```diff
@@ -242,10 +242,40 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 4. Promote your newest posts, pages, and products across your social media channels.
 
 == Changelog ==
-### 11.5-a.9 - 2022-10-19
+### 11.5-beta - 2022-10-25
+#### Enhancements
+- Dashboard: add a new illustration for WooCommerce recommendation.
+- Dashboard: add connection widget for unconnected sites.
+- Dashboard: add Jetpack Search Free to My products.
+- Dashboard: add Jetpack Social to My products.
+- Dashboard: ensure Apps card is always displayed, regardless of whether promotions are active.
+- Form block: add Contact Form child blocks to the Block Library.
+- Form block: add default spacing attributes on all form variations.
+- Form block: change layout flex styles.
+- Form block: improve parent block selection when inner block is already selected.
+- Form block: move Contact Forms child blocks to a new category and remove some Core blocks from the child blocks list.
+- Form block: remove duplicated contact form settings from the Contact Form block's toolbar in favor of the sidebar.
```

Comment:
Under bug fixes perhaps?

Reply:
It's not a bug really. Improved compatibility maybe but I think it works like this as well